docs/advertising/ad-customization.rst

```diff
@@ -28,7 +28,11 @@ The ad will be inserted into this container wherever this element is in the docu
 
 .. code-block:: html
 
-    <div id="ethical-ad-placement"></div>
+    <div data-ea-publisher="readthedocs"></div>
```

Comment:
I wonder what happens if users set this to something that isn't `readthedocs` -- it'll just fail and not show an ad right? :/

Reply:
Or the ads JS should re-set the values (publisher, keywords, etc.) on this element based on the response from the sustainability data API.